src/bridged_logging.rs

```diff
@@ -0,0 +1,245 @@
+// Copyright 2020, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+extern crate log as rs_log;
```

Comment:
It's weird to import a crate outside of the `lib.rs` file. What do you think @sdroege ?

Reply:
I don't see the problem :)